docs/website/docs/guides/deployment-configurations/cpu.md

```diff
@@ -104,12 +104,53 @@ drivers:
 ```console hl_lines="4 5"
 $ iree-run-module --list_drivers
 
-        cuda: CUDA (dynamic)
-  local-sync: Local execution using a lightweight inline synchronous queue
-  local-task: Local execution using the IREE multithreading task system
-      vulkan: Vulkan 1.x (dynamic)
+            cuda: NVIDIA CUDA HAL driver (via dylib)
+             hip: HIP HAL driver (via dylib)
+      local-sync: Local execution using a lightweight inline synchronous queue
+      local-task: Local execution using the IREE multithreading task system
+          vulkan: Vulkan 1.x (dynamic)
 ```
 
+You can download the runtime from a release or build from source.
+
+#### :octicons-package-16: Download the runtime from a release
+
+Python packages are regularly published to
+[PyPI](https://pypi.org/user/google-iree-pypi-deploy/). See the
+[Python Bindings](../../reference/bindings/python.md) page for more details.
+The `iree-runtime` package includes CPU support by via the `local-sync` and
+`local-task` drivers.
```

Comment:
Rather than duplicate the install instructions, how about we mention the `iree-runtime` and `iree-compiler` packages together?

What we want to communicate efficiently is:
* Packages for the compiler and runtime are available on pypi (stable) and github (nightly)
* We expect/encourage users to use the compiler packages, not build from source, for both the Python compiler APIs and the `iree-compile` tool
* We expect/encourage users to use the runtime _python bindings_ from packages
* Users can use `iree-run-module` and `iree-benchmark-module` from packages, but they will need to build the runtime from source for any C/C++ usage from a downstream application

I figured these "deployment configurations" would be focused on C/C++ linkage into an application, where a source build is needed. We do have projects like https://github.com/nod-ai/SHARK that use Python APIs from application code though.

Thoughts?

Reply:
Sounds good to me and a combined `Get the IREE compiler and runtime` section is fine for me. In that case we could restructure from

* Get the IREE compiler
  * Download the compiler from a release
  * Build the compiler from source
* Get the IREE runtime
  * Download the runtime from a release
  * Build the runtime from source

to

* Get the IREE compiler and Runtime
  * Download a release
  * Build from source
    * Build the compiler
    * Build the runtime

for example.

Reply:
Yeah, that's the high level view. The specifics can take a few iterations to get clear enough. I usually look at similar documentation for similar projects (PyTorch, ONNX, ONNX Runtime, TensorFlow, TensorFlow Lite, ncnn, etc.) for inspiration.